Source/Core/VideoBackends/D3D/PSTextureEncoder.cpp

```diff
@@ -147,12 +147,15 @@ size_t PSTextureEncoder::Encode(u8* dst, unsigned int dstFormat,
 
 		D3D::context->OMSetRenderTargets(1, &m_outRTV, nullptr);
 
-		ID3D11ShaderResourceView* pEFB = (srcFormat == PEControl::Z24) ?
-			FramebufferManager::GetEFBDepthTexture()->GetSRV() :
+		D3DTexture2D* efbTexture = (srcFormat == PEControl::Z24) ?
+			FramebufferManager::GetEFBDepthTexture() :
 			// FIXME: Instead of resolving EFB, it would be better to pick out a
 			// single sample from each pixel. The game may break if it isn't
 			// expecting the blurred edges around multisampled shapes.
-			FramebufferManager::GetResolvedEFBColorTexture()->GetSRV();
+			FramebufferManager::GetResolvedEFBColorTexture();
```

Comment:
The texture is resolved here, therefore it cannot be a Texture2DMSArray. We will need to find out why this call results in a multi sampled texture.